### ISSUE TITLE
Fix to use `ns` optional parameter in evaluating code

### ISSUE THREAD

**Issue by liquidz** [liquidz/vim-iced#225]
https://nrepl.org/nrepl/ops.html#_eval

**Comment by liquidz** [comment]
It works good in clojure project.
However, in figwheel-main, evaluating with `ns` parameter will raise the following error.

```
class java.lang.AssertionError
Execution error (AssertionError) at cljs.compiler/fn (compiler.cljc:491).
Assert failed: (ana/ast? sym)
```

I got no errors without `ns` parameter.
And also after evaluating `in-ns`, evaluating  some codes with `ns` parameter does not raise any errors.
weird.

**Comment by liquidz** [comment]
Closed by #380
